### ISSUE TITLE
Hosted version seems down?

### ISSUE THREAD

**Issue by sknebel** [fiatjaf/jekmentions#4]
Someone in #indiewebcamp who was looking at options just reported that it is broken, and I see a server error when trying it. Is that intended/known?

**Comment by fiatjaf** [comment]
Yes, it is known. I'm sorry. I was using a free Redis account from Rediscloud, but since there is absolutely zero usage, every three months they email me saying my account will be suspended if I don't use it. This time I let it go.

What do you think would be a good solution to this case? What should I do? I really don't know.

**Comment by sknebel** [comment]
IMHO it makes no sense to keep it online if it has no users. Just update the repo description and references on indiewebcamp.com (I can do those if you want) to make it clear (including if you consider the entire project abandoned or not).

**Comment by sknebel** [comment]
I updated the Indiewebcamp wiki and marked the service as shutdown (https://indiewebcamp.com/wiki/index.php?diff=25684&oldid=19167&rcid=25646).
